FlowCrypt/Common UI/AttachmentManager.swift

```diff
@@ -45,15 +45,21 @@ final class AttachmentManager: NSObject {
 }
 
 extension AttachmentManager: AttachmentManagerType {
-    func open(_ attachment: MessageAttachment) {
-        filesManager.saveToFilesApp(file: attachment, from: self)
-            .sinkFuture(
-                receiveValue: {},
-                receiveError: { [weak self] error in
-                    self?.handle(error)
-                }
-            )
-            .store(in: &self.cancellable)
+    func download(_ file: FileType) {
+        Task {
+            do {
+                let url = try await filesManager.save(file: file)
+                openDocumentsController(from: url)
+            } catch {
+                handle(error)
+            }
+        }
+    }
+
+    @MainActor func openDocumentsController(from url: URL) {
+        let documentController = UIDocumentPickerViewController(forExporting: [url])
+        documentController.delegate = self
+        present(documentController, animated: true, completion: nil)
     }
 
     private func handle(_ error: Error) {
```

Comment:
This method no longer has any value and can be removed. Wherever this is used, call `await controller?.showToast("\("message_attachment_saved_with_error".localized) \(error.localizedDescription)")` directly instead.